ElectronicObserver/Window/Wpf/Fleet/ViewModels/FleetStateViewModel.cs

```diff
@@ -407,14 +407,12 @@ public void RefreshFleetState()
 
 		}
 	}
-	public int GetIconIndex()
+	public IconContent GetIconIndex()
```

Comment:
Should be renamed GetIcon since we're not getting the index anymore ?

Reply:
Yeah, makes sense.
Wonder if the IconContent enum should be IconId or something instead, think that would be clearer.